### ISSUE TITLE
RasaX UI breaks on refresh page, test user conversations never load

### ISSUE THREAD

**Issue by ActuallyAcey** [RasaHQ/rasa#5326]
**Rasa version**:
1.7.1

**Rasa SDK version** (if used & relevant):
Not relevant, probably

**Rasa X version** (if used & relevant):
0.25.2

**Python version**:
3.7.4

**Operating system** (windows, osx, ...):
Ubuntu 18.04 LTS / Windows 10

**Issue**:
After RasaX is fully up and running, you can share your bot to test users. Accessing the URL and talking to the bot is just fine, but when you return to the dashboard, many elements of the UI disappear completely. 

BEFORE
![image](https://user-images.githubusercontent.com/44527014/75424976-bfa53400-5967-11ea-9af8-4ac9bf0a5d26.png)

AFTER TEST USER HAS A CONVERSATION AND PAGE IS REFRESHED
![image](https://user-images.githubusercontent.com/44527014/75425015-d9df1200-5967-11ea-9011-62ad5102c6fe.png)
**Error (including full traceback)**:

A quick look into Chrome's developer console shows these errors:
![image](https://user-images.githubusercontent.com/44527014/75425063-f8dda400-5967-11ea-9beb-cb83ba9c2c71.png)

Similarly, accessing the URLs posted in the errors shows this message:
```{"reasons":["Authorization header not present."],"exception":"Unauthorized"}```

It's worth noting that logging out and logging back into RasaX fixes the issue temporarily, but it breaks again the moment you refresh the page. 

At **no point** does the actual conversation with the test user actually show up in the Dashboard. However, the user messages do show up in the **Annotate new data** section. 

**Command or request that led to error**:
```
Conversations never load, refreshing the page breaks the UI. 
```

**Content of configuration file (config.yml)** (if relevant):
```yml
Not relevant.
```

**Content of domain file (domain.yml)** (if relevant):
```yml
Not relevant.
```

**Comment by erohmensing** [comment]
Hi @ActuallyAcey, thanks for pointing this out. It's an issue due to the different permissions of test users and regular users, and should be fixed in the next minor release. 

> It's worth noting that logging out and logging back into RasaX fixes the issue temporarily, but it breaks again the moment you refresh the page.
After you've logged out and back in, do you still have the test chat window upon when you refresh the page and hit the problem again?

**Comment by ActuallyAcey** [comment]
> After you've logged out and back in, do you still have the test chat window upon when you refresh the page and hit the problem again?

It seems keeping the test chat window open doesn't affect whether the UI breaks or not. The UI breaking also seems inconsistent; it sometimes still works after logging back in and refreshing.

**Comment by erohmensing** [comment]
Interesting. We have been able to track down the issue itself, but I'm not sure about the inconsistency with after logging out and back in. Thanks for noticing that, we can make sure to try to test that a few different times to see if the issue arises again after the fix.

**Comment by erohmensing** [comment]
> It's an issue due to the different permissions of test users and regular users

Fixed in 0.26.0: https://rasa.com/docs/rasa-x/changelog/rasa-x-changelog/#id175
